5_databases/week2/homework

```diff
@@ -0,0 +1,145 @@
+INSERT INTO task (title, description, created, updated, due_date, status_id)
+VALUES ('Learn an  instrument', 'Maybe violin', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP, '2023-12-31', 1);
+
+
+UPDATE task
+SET title = 'Learn an game'
+WHERE title = 'Learn an  instrument';
+
+
+
+UPDATE task
+SET due_date = '2023-01-01 00:00:00'
+WHERE id = 1;
+
+
+
+UPDATE task
+SET status_id = '2'
+WHERE id = "1";
+
+
+UPDATE task
```

Comment:
Cool solution but you can keep it simple 
`
UPDATE task
SET status_id = 3
WHERE id = 1;`